src/ui/services (business)/products/add-new-product.ui-service.ts

```diff
@@ -0,0 +1,36 @@
+import { expect, Page } from "@playwright/test";
+import { apiConfig } from "config/api-config";
+import { STATUS_CODES } from "data/statusCodes";
+import { ProductsPage } from "ui/pages/products/products.page";
+import { AddNewProductPage } from "ui/pages/products/add-new-product.page";
+
+import _ from "lodash";
+import { IProduct, IProductResponse } from "types/products.type";
+import { generateProductData } from "data/products/generateProductData";
+
+export class AddNewProductUiService {
+  private productsPage: ProductsPage;
+  private addNewProductPage: AddNewProductPage;
+  constructor(private page: Page) {
+    this.addNewProductPage = new AddNewProductPage(page);
+    this.productsPage = new ProductsPage(page);
+  }
```

Comment:
Вариант, сделать базовый сервис в котором будет сущность `Page` и в последующих сервисах получим следующее (или просто даже `PageHolder`, его можно даже будет для `page` классов использовать) 
```suggestion
export class AddNewProductUiService extend PageHolder {
  private productsPage: ProductsPage = new AddNewProductPage(this.page)
  private addNewProductPage: AddNewProductPage = new ProductsPage(this.page);
```